.gitignore

```diff
@@ -1,4 +1,4 @@
-clboss
+/clboss
```

Comment:
What does this change do?

Reply:
We're trying to ignore the clboss binary, but I added a `contrib/clboss` directory (for python library stuff) and it was unintentionally also ignored.  Change makes it only ignore the top-level binary ...

Reply:
Ah cool, thanks!